platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -207,17 +198,9 @@ public void uninstallFeature(String feature) {
     LOGGER.trace("Uninstalling feature {}", feature);
 
     try {
-      AccessController.doPrivileged(
-          (PrivilegedExceptionAction<Void>)
-              () -> {
-                featuresService.uninstallFeature(
-                    feature, EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
-                return null;
-              });
+      syncInstaller.uninstallFeatures(feature);
```

Comment:
The `ProfileInstallCommand` should also be updated to use the syncInstaller as it is used for automated installs. Might even be more important than here.

Reply:
Let's open the ticket for it so we don't forget 😉